### PR TITLE
Fix broken links, typos, grammar, and add site button

### PR DESCRIPTION
**Pull request by simonri**
### Overview
- Update broken links
- Replace missing social media
- Fix typos and grammar in `CONTRIBUTING.md`, `README.md`, and `header.html`.
- Fix "add site" button

### Changed sites:
- Adultiful (dead)
- HoneySexDoll (dead)
- AmateurCanada (dead)
- AbrahamArtist (dead)
- AeroFS (change name to Redbooth)
- Pogoplug (shut down 2016)
- Hyper.sh (shut down [Jan 15](https://news.ycombinator.com/item?id=18734658))
- Lighthouse.cash (dead since late 2018)
- Coinbought (dead)
- CZIP (dead)
- Findbits (shut down 2019)
- Taskopus (shut down [2 weeks ago](https://www.reddit.com/r/Taskopus/comments/c22syf/big_news_taskopus_is_shutting_down/))
- pairNIC (changed name to pairDOMAINS)
- cheddr (shut down Mar 01 by the government)
- CashVid (shut down 3 months ago due to cannabis promotion)
- Keyport (dead)
- City Lounge (dead)
- Dunkin' Donuts (shut down due to bankruptcy)
- Flavours Place (dead)

### CODE REVIEW

File: _data/accommodation.yml [hunk 1]
```diff
@@ -10,8 +10,8 @@ websites:
   country: DE
   doc: https://www.aohostels.com/en/services/frequent-guest-advantage/
   keywords:
-  - bitpay
-  - bip70
+    - bitpay
```
Comment (kenman345) [inline _data/accommodation.yml]: the spacing was proper on these. I used Ruby to reorder them and spit back out updated YAML files so I know that the spacing was right before. Why did you feel it needing changing and not everywhere?

**Comment by kenman345** [comment]
Dunkin Donuts has not shut down due to bankruptcy. What are you trying to pull with this?

**Comment by kenman345** [comment]
Adding a Do not merge label until we can confirm every change in this PR @EchterAgo

File: _data/accommodation.yml (hunk 1 above)
Comment (simonri) [reply _data/accommodation.yml]: I forgot to disable my "Prettier" plugin during the first commit so it automatically got changed when I saved the file. The tabs are reverted in the "Revert tab from first commit in accommodation.yml" commit.

**Comment by simonri** [comment]
> Dunkin Donuts has not shut down due to bankruptcy. What are you trying to pull with this?

I can confirm that was a mistake from me. They have shut down in Sweden so when you go to the site from a Swedish IP it says that they have shut down.

File: _data/accommodation.yml (hunk 1 above)
Comment (kenman345) [reply _data/accommodation.yml]: Thank you for your contribution. Please make another revert though as this page would say my comment was out of date if it were not still having that additional spacing in a few spots on this YML file.

**Comment by kenman345** [comment]
Thank you so much for contributing. I added some comments of things that need to be fixed before we would be okay merging this into our master branch. Please fix the spacing issue and remove that now orphaned icons where they exist.

**Comment by kenman345** [comment]
Also wanted to add I merged in a change to art.yml so dont be surprised if it might come up that you're behind/conflict needs to be resolved for the PR, though the change was a new record in the art.yml file and your removal shouldnt hit any of the same lines. Just wanted to give you the heads up

**Comment by simonri** [comment]
Alright I think I've fixed everything you asked for now.

File: _data/cryptoservices.yml [hunk 2]
```diff
@@ -661,19 +633,6 @@ websites:
   bch: true
   btc: false
   othercrypto: false
-- name: Taskopus
```
Comment (EchterAgo) [inline _data/cryptoservices.yml]: Hmmm, its gone already? I'll add all the removed entries to my list to check again later.

File: _data/donation-human-needs.yml [hunk 3]
```diff
@@ -44,10 +44,9 @@ websites:
   twitter: FareShare1
   region: oc
   country: AU
-  bch: true
-  btc: true
-  othercrypto: true
-  doc: https://twitter.com/FareShare1/status/949041934453981185
+  bch: false
+  btc: false
+  othercrypto: false
```
Comment (EchterAgo) [inline _data/donation-human-needs.yml]: Hmm, they still have 5 BCH in their donation address:

https://blockchair.com/bitcoin-cash/address/3JLehRBNg1oH4tyCZ4Jy1h1e3DwZzU5Nqm

File: _data/donation-non-profit.yml [hunk 4]
```diff
@@ -205,14 +205,14 @@ websites:
   othercrypto: true
   doc: https://www.sens.org/donate
 - name: The Bitcoin Cash Fund
-  url: https://www.thebitcoincash.fund/
+  url: https://www.bitcoincash.org/
```
Comment (EchterAgo) [inline _data/donation-non-profit.yml]: This is wrong, the new site is https://bitcoincashers.org/

**Comment by georgengelmann** [comment]
This pull request needs some fixes and rebase

**Comment by kenman345** [comment]
Can this get some love? Otherwise I'll be making some of the same changes and opening a fresh PR within the next week.

**Comment by zquestz** [comment]
This PR is being closed. The cleanup PR is already up!

**Pull request closed without merging**